### DISCUSSION TITLE
Dashboards (and map) empty

### DISCUSSION THREAD

**Discussion by david-77** [teslamate-org/teslamate#2992] [Getting started] [answered]
Hi,

I set up Teslamate with dockercompose file, I have my two urls set (Teslamate webUI & Grafana) working, but the is nothing interesting in the first page, and dashboards are empty  : 
![image](https://user-images.githubusercontent.com/46008203/203571037-f26043b0-2f2e-4130-ba62-e908aca743c1.png)
And here is an example of a dashboard link : 
![image](https://user-images.githubusercontent.com/46008203/203571224-aeb66bd4-6d94-47da-9561-dc585f1cff80.png)
There seems to have query errors  (500) : 
![image](https://user-images.githubusercontent.com/46008203/203571322-62b40e9f-3756-4de8-bd17-d61f9349492c.png)

Is there something I missed in configuration  ?
Thanks

**Comment by DrMichael** [comment]
Which reverse proxy are you using? Did you put the URL's in Réglages?

**Reply by david-77** [reply]
I am not using any reverse proxy (as far as I know). . And yes, the URL in Reglage is fine.
For info, my docker containers run on my Rasberry PI, and I do not have any Internet access problem with other containers.
I drove yesterday and now the map displays correctly in Teslamate homepage. It seems the car needs to be started after Teslamate installation.
But the issue remains for the dashboards (still empty/500 error code)

**Comment by DrMichael** [comment, marked as answer]
What is the error, when you hover over the red triangles? What gives "docker-compose grafana logs -f"?

**Reply by david-77** [reply]
Thanks for the tip.
I checked the logs and it was pretty clear ; the database password was considered as an "invalid option" un grafana's http requests.
My docker compose had exactly the same password for the 3 services.
As usual, I put my password between quotation marks, and it appears that grafana's image does not handle it properly.
I removed the quotation marks in all 3 services using it, and removed previous created containers and volumes, and... tada : it works ;)
Thanks !
